package.json

```diff
@@ -1,9 +1,10 @@
 {
-  "name": "ts-framework",
-  "version": "0.3.0",
+  "name": "tsframework-full",
```

Comment:
I intend to rename the whole thing without the -.
That way we can have tsframework-component naming convention.
I find it more easy to understand which component it is (here it's the full framework for example).